google/cloud/sql/connector/refresh_utils.py

```diff
@@ -150,8 +150,9 @@ async def _get_ephemeral(
         request = google.auth.transport.requests.Request()
         credentials.refresh(request)
 
+    stripped_token = credentials.token.rstrip(".")
```

Comment:
Same thing here: a comment or something to remind us to remove this? Perhaps an issue to remove this once we fix it?